src/App.jsx

```diff
@@ -1,15 +1,25 @@
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import { ThemeProvider } from "@mui/material";
 import { theme } from "./Provider/ThemeProvider";
-import Routes from "./Routes";
+import ProtectedRoutes from "./Routes/ProtectedRoute";
+import SigninPage from "./pages/Signin/SigninPage";
+import HomePage from "./pages/HomePage/HomePage";
+import ForgotPasswordPage from "./pages/ForgotPassword/ForgotPasswordPage"
+import HorizontalLinearStepper from "./pages/RestaurantCreation/RestaurantCreationStepper";
 
 function App() {
-
   return (
     <>
       <ThemeProvider theme={theme}>
         <BrowserRouter>
-          <Routes />
+        <Routes>
+          <Route path="/" element={<SigninPage />} />
+          <Route path="forgot-password" element={<ForgotPasswordPage />} />
+          <Route element={<ProtectedRoutes />}>
+            <Route path="/home" element={<HomePage />} />
+            <Route path="/create-restaurant" element={<HorizontalLinearStepper />} />
+          </Route>
+        </Routes>
         </BrowserRouter>
       </ThemeProvider>
```

Comment:
The PR is super huge. and the BUG is not clear what you are trying to fix.
Have a proper PR description